src/main/webapp/WEB-INF/tags/instructor/results/filterEditModal.tag

```diff
@@ -0,0 +1,116 @@
+                        <%=Const.NO_SPECIFIC_RECIPIENT%>
+                      </option>
+                    </select>
+                  </div>
+                </div>
+              </c:if>
+            </div>
+            <div class="col-md-5">
+              <label class="control-label">
+                Additional settings:
+              </label>
+              <div data-toggle="tooltip" title="Group results in the current view by team">
+                <div class="checkbox">
+                  <label<c:if test="${filterPanel.sortType == 'question'}"> class="text-strike"</c:if>>
+                    <input type="checkbox" name="<%=Const.ParamsNames.FEEDBACK_RESULTS_GROUPBYTEAM%>"
+                           id="<%=Const.ParamsNames.FEEDBACK_RESULTS_GROUPBYTEAM%>"
+                           <c:if test="${filterPanel.groupedByTeam}">checked</c:if>
+                           <c:if test="${filterPanel.sortType != 'question'}">class="checkbox-group-by-team"</c:if>> Group by Teams
+                  </label>
+                </div>
+              </div>
+              <div data-toggle="tooltip" title="Show statistics">
+                <div class="checkbox">
+                  <label<c:if test="${filterPanel.sortType == 'recipient-giver-question' or filterPanel.sortType == 'giver-recipient-question'}"> class="text-strike"</c:if>>
+                    <input type="checkbox" id="show-stats-checkbox"
+                           name="<%=Const.ParamsNames.FEEDBACK_RESULTS_SHOWSTATS%>"
+                    <c:if test="${filterPanel.statsShown}"> checked</c:if>> Show Statistics
+                  </label>
+                </div>
+              </div>
+              <div data-toggle="tooltip" title="Indicate missing responses">
+                <div class="checkbox">
+                  <input type="checkbox"
+                         id="indicate-missing-responses-checkbox"
+                         value="true"
+                         name="<%=Const.ParamsNames.FEEDBACK_RESULTS_INDICATE_MISSING_RESPONSES%>"
+                  <c:if test="${filterPanel.missingResponsesShown}"> checked</c:if>>
+                  Indicate Missing Responses
+                </div>
+              </div>
+            </div>
+          </div>
+
+          <input type="hidden" name="<%=Const.ParamsNames.FEEDBACK_SESSION_NAME%>"
+                 value="${filterPanel.feedbackSessionName}">
+          <input type="hidden" name="<%=Const.ParamsNames.COURSE_ID%>"
+                 value="${filterPanel.courseId}">
+          <input type="hidden" name="<%=Const.ParamsNames.USER_ID%>"
+                 value="${data.account.googleId}">
+
+          <div class="modal-footer">
+            <button id="submitBtn" type="submit" class="btn btn-primary pull-left">Change View</button>
+            <button type="button" class="btn btn-danger pull-right" data-dismiss="modal">Cancel</button>
+          </div>
```

Comment:
Just realised another problem: `modal-footer` should not be inside `modal-body`, but directly inside `modal-content` (take a look at how the other modals are done e.g. the bootbox one). Placing `modal-footer` inside `modal-body` causes the buttons not to align to the content.

Also, just realised that `modal-footer` already aligns content to the right, so `pull-right` can be dropped entirely. For consistency with the confirm publish/unpublish modal, the cancel button should be on the left of the confirm button, with both buttons together side-by-side on the bottom right corner of the modal. Hence, drop `pull-left` too.

Summary: place this HTML directly inside the `modal-content`, after `modal-body` (i.e., after line 113 in the current version):
```HTML
<div class="modal-footer">
  <button type="button" class="btn btn-danger pull-right" data-dismiss="modal">Cancel</button>
  <button id="submitBtn" type="submit" class="btn btn-primary pull-left">Change View</button>
</div>
```

The result should look like this:
<img width="615" alt="screen shot 2017-07-27 at 6 33 47 pm" src="https://user-images.githubusercontent.com/6811428/28666505-2f6e7136-72fa-11e7-865c-8499c433be1d.png">